### DISCUSSION TITLE
saf-training/courses/user/15

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#268] [Ideas]
# saf-training/courses/user/15

15.1 Take the Class Survey Take our 9 question SAF User Class survey to give feedback to fuel class improvement. 15.2 Take the Beginner Security Automation Developer Class This ...

https://mitre.github.io/saf-training/courses/user/15.html

**Comment by karikarshivani** [comment]
Things that people may like notifications for:
- New STIGs that we're interested in updating to
- New profiles (To Do / In Progress / Complete)
- Newer versions of existing profiles (To Do / In Progress / Complete)
- Hardening content (To Do / In Progress / Complete)
- Converters (To Do / In Progress / Complete)
